README.md

```diff
@@ -49,17 +49,16 @@ def test_request_to_mock_server(mock_server):
 
     with mock_server(imposter):
         # Make request to mock server - exercise code under test here
-        response = requests.get("{}/test".format(imposter.url))
```

Comment:
I'm keeping everything Python 3.5 compatible for now.

Reply:
ok, reverted